src/components/GeneralInfo/SystemCard/SystemCard.js

```diff
@@ -152,6 +154,7 @@ class SystemCardCore extends Component {
                       </Fragment>
                     ),
                     size: 'md',
+                    customClass: 'sentry-mask data-hj-suppress',
```

Comment:
Just because you are using it multiple times, I would consider adding those className combination to the constants.js file 
The name of the constant could be "HIDE_CLASSES" or something similar or better suited for the reason those values are being hidden for
Just an idea of course if you want 😇  I don't think it is a must, but it could improve readability of the code in my opinion 😀

Reply:
I dont mind this idea, but I think it would be better implemented if we had that in something like the FEC repo and then imported the class since we use this in a good amount of repos.